src/overlap_check/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+message("")
+
+set(LINK_LIBS dagmc)
+set(LINK_LIBS_EXTERN_NAMES)
+
+find_package(OpenMP)
+
+add_subdirectory(build)
```

Comment:
Maybe this directory should be `cli` or something more specific? (Although I'm not convinced that it needs its own subdirectory?)

Reply:
The paradigm we've been using for the rest of the stuff we build in DAGMC is to have executables built in a subdirectory called `build`

Reply:
I see - I guess I haven't paid much attention to this before - is there a reason to avoid doing it in the same directory?

Reply:
Not that I'm aware of. I'd be on board with that model.